rust/src/composepost.rs

```diff
@@ -980,6 +981,10 @@ fn rewrite_rpmdb_for_target_inner(rootfs_dfd: &openat::Dir) -> Result<()> {
     let mut dbfd = Rc::try_unwrap(dbfd).unwrap();
     dbfd.seek(std::io::SeekFrom::Start(0))?;
 
+    // In the interests of build stability, rewrite the INSTALLTIME and INSTALLTID tags
+    // to match SOURCE_DATE_EPOCH if it's present
+    normalization::rewrite_rpmdb_timestamps(&mut dbfd)?;
```

Comment:
For now, I'd be much more comfortable landing this if it was gated behind a treefile option.

Something like:

```
diff --git a/docs/treefile.md b/docs/treefile.md
index 35648bcd..8560ba51 100644
--- a/docs/treefile.md
+++ b/docs/treefile.md
@@ -111,6 +111,10 @@ It supports the following parameters:
     library in the target filesystem tree understands.  However, this is
     a relatively new default, so the value `host` is provided as a fallback
 
+ * `rpmdb-normalize`: boolean, optional.  Defaults to `false`.  If enabled,
+    this runs through code which filters out potentially uninitialized data
+    from the RPM database.  Somewhat experimental.
+
  * `cliwrap`: boolean, optional.  Defaults to `false`.  If enabled,
     rpm-ostree will replace binaries such as `/usr/bin/rpm` with
     wrappers that intercept unsafe operations, or adjust functionality.
diff --git a/rust/src/treefile.rs b/rust/src/treefile.rs
index c996bdac..b5af89ea 100644
--- a/rust/src/treefile.rs
+++ b/rust/src/treefile.rs
@@ -382,7 +382,8 @@ fn treefile_merge(dest: &mut TreeComposeConfig, src: &mut TreeComposeConfig) {
         preserve_passwd,
         check_passwd,
         check_groups,
-        postprocess_script
+        postprocess_script,
+        rpmdb_normalize
     );
     merge_hashsets!(ignore_removed_groups, ignore_removed_users);
     merge_maps!(add_commit_metadata);
@@ -1282,6 +1283,9 @@ pub(crate) struct TreeComposeConfig {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub(crate) rpmdb: Option<RpmdbBackend>,
 
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub(crate) rpmdb_normalize: Option<bool>,
+
     #[serde(flatten)]
     pub(crate) legacy_fields: LegacyTreeComposeConfigFields,
 
```

?

And then key off that boolean here?